### ISSUE TITLE
Issue while running demo.py "Creating layer proposal"

### ISSUE THREAD

**Issue by vinayakarannil** [tianzhi0549/CTPN#116]
I have built caffe succesfully and while running demo.py the below error is coming and getting core dumped

I0621 16:58:41.372174  2732 layer_factory.hpp:76] Creating layer proposal
F0621 16:58:41.372180  2732 layer_factory.hpp:80] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Python (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, LRN, Log, Lstm, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Pooling, Power, ROIPooling, ReLU, Reduction, Reshape, Reverse, SPP, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, SmoothL1Loss, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, Transpose, WindowData)
*** Check failure stack trace: ***
Aborted (core dumped)

**Comment by tianzhi0549** [comment]
@vinayakkailas Did you set `WITH_PYTHON_LAYER := 1` in `Makefile.config`?

**Comment by vinayakarannil** [comment]
Yes that was the issue. Fixed by making WITH_PYTHON_LAYER =1.
